File: Buiers/Controllers/HomeController.vb
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        Return View()
    End Function

    Function About() As ActionResult
        ViewData("Message") = "Your application description page."

        Return View()
    End Function

    Function Contact() As ActionResult
        ViewData("Message") = "Your contact page."

        Return View()
    End Function

    Function FAQ() As ActionResult
        ViewData("Message") = "Your FAQ page."

        Return View()
    End Function

    Function Properties() As ActionResult
        ViewData("Message") = "Your Properties page."

        Return View()
    End Function

    Function Detailed() As ActionResult
        ViewData("Message") = "Detailed property page."

        Return View()
    End Function

    Function Buyers() As ActionResult
        ViewData("Message") = "Buyers guide page."

        Return View()
    End Function

    Function Sellers() As ActionResult
        ViewData("Message") = "Sellers guide page."

        Return View()
    End Function

    Function Home() As ActionResult
        ViewData("Message") = "Home page."

        Return View()
    End Function

    Function ThankYou() As ActionResult
        ViewData("Message") = "Thank you for your submission."

        Return View()
    End Function

    Function MyProperties() As ActionResult
        ViewData("Message") = "My properties page."

        Return View()
    End Function

    Function AddProperty() As ActionResult
        ViewData("Message") = "Add your property page."

        Return View()
    End Function

    Function Register() As ActionResult
        ViewData("Message") = "Account Registration page."

        Return View()
    End Function

End Class

File: Auctionprop08052020/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace Auctionprop08052020.Controllers
{
    public class Homecontroller : Controller
    {
        public ActionResult Index()
        {
            return View();
        }


        public ActionResult Properties()
        {
            return View();
        }

        public ActionResult Buyers()
        {
            ViewBag.Message = "Your application description page.";

            return View();
        }

        public ActionResult Sellers()
        {
            ViewBag.Message = "Your application description page.";

            return View();
        }

        public ActionResult Contact()
        {
            ViewBag.Message = "Your application description page.";

            return View();
        }

        public ActionResult FAQ()
        {
            ViewBag.Message = "Your contact page.";

            return View();
        }

        public ActionResult About()
        {
            ViewBag.Message = "Your contact page.";

            return View();
        }

        public ActionResult Add()
        {
            ViewBag.Message = "Add property";

            return View();
        }
        public ActionResult Error()
        {
            ViewBag.Message = "404 Error Page";

            return View();
        }
        public ActionResult Privacy()
        {
            ViewBag.Message = "Privacy Policy";

            return View();
        }
        public ActionResult Forbidden()
        {
            ViewBag.Message = "403 Forbidden Page";

            return View();
        }
        public ActionResult Terms()
        {
            ViewBag.Message = "Terms & Conditions Page";

            return View();
        }
    }
}
